components/ImageGallery/ImageGalleryBase.js

```diff
@@ -29,11 +29,15 @@ export class ImageGalleryBase extends PureComponent {
 
     autoBindReact(this);
 
+    const { selectedIndex } = props;
+
     this.state = {
-      selectedIndex: this.props.selectedIndex || 0,
+      selectedIndex,
```

Comment:
You sure we don't need fallback `0` here?
If we do, please add it to `defaultProps`

Reply:
Already done in previous PR in chain where defaultProps are defined.